svelte/lib/profil/Profil.svelte

```diff
@@ -0,0 +1,279 @@
+<script lang="ts">
+  import type { Departement, Utilisateur } from './profil.d';
+  import ChampTexte from '../ui/ChampTexte.svelte';
+  import SelectionDomaineSpecialite from '../inscription/SelectionDomaineSpecialite.svelte';
+  import Formulaire from '../ui/Formulaire.svelte';
+  import SelectionDepartement from '../inscription/SelectionDepartement.svelte';
+  import SelectionOrganisation from '../inscription/SelectionOrganisation.svelte';
+  import type {
+    EstimationNombreServices,
+    Organisation,
+  } from '../inscription/inscription.d';
+  import SelectionNombreServices from '../inscription/SelectionNombreServices.svelte';
+  import Bouton from '../ui/Bouton.svelte';
+
+  // eslint-disable-next-line svelte/valid-compile
+  export let departements: Departement[];
+  export let utilisateur: Utilisateur;
+  export let entite: Organisation;
+  export let estimationNombreServices: EstimationNombreServices[];
+
+  const modeleTelephone = '^0\\d{9}$';
+  let departement: Departement | undefined = departements.find(
+    (d) => d.code === entite.departement
+  );
+  let nombreServices: string = `${utilisateur.estimationNombreServices.borneBasse}_${utilisateur.estimationNombreServices.borneHaute}`;
+  $: {
+    utilisateur.estimationNombreServices = {
+      borneBasse: nombreServices.split('_')[0],
+      borneHaute: nombreServices.split('_')[1],
+    };
+  }
+
+  let formulaire: Formulaire;
+
+  const valide = async () => {
+    if (formulaire.estValide()) {
+      await axios.put('/api/utilisateur', {
+        ...utilisateur,
+        siretEntite: entite.siret,
+      });
+      window.location.href = '/tableauDeBord';
+    }
+  };
+</script>
+
+<div class="contenu-profil">
+  <div>
+    <h1>Mes informations MonServiceSecurisé</h1>
+    <h2>
+      Informations recueillies dans le cadre de votre inscription à
+      MonServiceSécurisé.
+    </h2>
+  </div>
+
+  <Formulaire classe="formulaire-profil" bind:this={formulaire}>
+    <div class="bloc">
+      <div>
+        <div class="info-champ-obligatoire requis">Champ obligatoire</div>
+        <h3>Mon identité</h3>
+      </div>
+      <span>Mail professionnel : <b>{utilisateur.email}</b></span>
+      <div class="champ">
+        <label class="requis" for="prenom">Prénom</label>
+        <ChampTexte
+          id="prenom"
+          nom="prenom"
+          bind:valeur={utilisateur.prenom}
+          aideSaisie="ex : Jean"
+          requis
+          messageErreur="Le prénom est obligatoire. Les chiffres ne sont pas autorisés."
+        />
+      </div>
+      <div class="champ">
+        <label class="requis" for="nom">Nom</label>
+        <ChampTexte
+          id="nom"
+          nom="nom"
+          bind:valeur={utilisateur.nom}
+          aideSaisie="ex : Dupont"
+          requis
+          messageErreur="Le nom est obligatoire. Les chiffres ne sont pas autorisés."
+        />
+      </div>
+      <div class="champ">
+        <label class="requis" for="domaine-specialite"
```

Comment:
Le `for` ne semble pas fonctionner. Tu pourras vérifier ?

Reply:
Corrigé :heavy_check_mark: